sound/soc/sof/ipc3-topology.c

```diff
@@ -1594,6 +1594,17 @@ static int sof_ipc3_widget_setup_comp_dai(struct snd_sof_widget *swidget)
 	if (ret < 0)
 		goto free;
 
+	/* DAI 2 is used as the first DAI for SDW in topology, normalize it to 0 */
+	if (comp_dai->type == SOF_DAI_INTEL_ALH) {
+		comp_dai->dai_index -= 2;
+		if (comp_dai->dai_index < 0) {
+			dev_err(sdev->dev,
+				"Invalid ALH dai index %d, Please use SDW Pin 2 and above\n",
```

Comment:
maybe we can say that only Pin numbers > 1 are supported instead of saying please use Pin 2

Reply:
> maybe we can say that only Pin numbers > 1 are supported instead of saying please use Pin 2

Updated